microsoft-edge/extensions-chromium/publish/create-dev-account.md

```diff
@@ -25,7 +25,11 @@ If you don't have an account or if you have an existing commercial account with
 If you have a developer account registered on Partner center, use the corresponding Microsoft account \(MSA\) to sign into your developer account, and then enroll in the Microsoft Edge program.  
 
 > [!NOTE]
-> Today the Microsoft Edge extensions does not support signing up with a work or school account.  In the future, the Microsoft Edge extensions team plans to support linking Azure AD tenants to MSA accounts for extension management.  
+>  You can choose to link your work account on Partner center to manage your Edge extensions. For this, you first need to register to Partner center using a Microsoft account. To link your work account, navigate to **Tenants** under **Account settings** and select either **Associate your Azure AD account with Partner center** or **Create a new Azure AD**.
```

Comment:
Making some changes to the sentence here. Should we bullet this out in 2 or more steps?

You can link your work account on Partner Center to manage your Edge extensions. For this, you first need to register to the Microsoft Edge program on the Partner Center using a Microsoft Personal account. Follow the steps mentioned here <link> to register, if you haven't already done so.  Once registered, navigate to **Tenants** under **Account settings** and select either **Associate your Azure AD account with Partner center** or **Create a new Azure AD**.